src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
```

Comment:
Don't use verbs for class/interface names. Consider renaming `IsoscelesTrapezoid` to something that does not start with a verb. For example, `TrapezoidIsosceles` could be a better name.